src/index.html

```diff
@@ -4,14 +4,160 @@
+      <fieldset class="form__field">
+        <legend>Personal information</legend>
+        <div class="form__div">
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            name="Surname"
+            id="surname"
+          >
+        </div>
+        <div class="form__div">
+          <label for="name"> Name:</label>
+          <input
+            type="text"
+            name="Name"
+            id="name"
+            minlength="2"
+          >
+        </div>
+        <div class="form__div">
+          <label for="number">How old are you?</label>
+          <input
+            type="number"
+            value="12"
+            name="age"
+            id="number"
+            max="100"
+            min="0"
+          >
+        </div>
+        <div class="form__div">
+          <label for="birth">Full date of birth:</label>
+          <input
+            type="date"
+            name="birth"
+            id="birth"
+            >
+        </div>
+        <div class="form__div2">
+          <label for="acceping">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            name="accept"
+            id="acceping"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="form__field">
+        <legend>Registratio</legend>
+        <div class="form__div">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            name="E-mail"
+            placeholder="email@example.com"
+            id="email"
+          >
+        </div>
+        <div class="form__div2">
+          <label for="password">Pasword:</label>
+          <input
+            type="password"
+            name="password"
+            id="password"
+            required
+            maxlength="20"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="form__field">
+        <legend>An interesting fact about you!</legend>
+        <div class="form__div">
+          <label for="pet">Do you love cats?</label>
+          <input
+              type="radio"
+              name="cats"
+              id="pet"
+              >
+                Yes
+          <input
+              type="radio"
+              name="cats"
+            >
+                No
+          </label>
+        </div>
+        <div class="form__div">
+          <label for="color">What is your favorite color?</label>
+          <input
+              type="color"
+              name="color"
+              id="color"
+            >
+        </div>
+        <div class="form__div">
+          <label for="sleep"> What time do you go to bed?</label>
+          <input
+              type="time"
+              name="sleep"
+              id="sleep"
+            >
+        </div>
+        <div class="form__div">
+          <label for="BrandsCar">Whar are your favorite brand of cars?</label>
+          <select
+                name="Brands"
+                id="BrandsCar"
```

Comment:
id should not start with Capital letter
```suggestion
                id="brandsCar"
```